mteb/tasks/Classification/eng/LegalBenchClassification.py

```diff
@@ -2916,3 +2916,528 @@ def dataset_transform(self):
+
+
+class Diversity1LegalBenchClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="Diversity1LegalBenchClassification",
+        description="Given a set of facts about the citizenships of plaintiffs and defendants and the amounts associated with claims, determine if the criteria for diversity jurisdiction have been met (variant 1).",
+        reference="https://huggingface.co/datasets/nguha/legalbench",
+        dataset={
+            "path": "nguha/legalbench",
+            "name": "diversity_1",
+            "revision": "12ca3b695563788fead87a982ad1a068284413f4",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2000-01-01", "2023-08-23"),  # best guess
+        form=["written"],
+        domains=["Legal"],
+        task_subtypes=[],
+        license="cc-by-4.0",
+        socioeconomic_status="high",
+        annotations_creators="expert-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+        @misc{guha2023legalbench,
+            title={LegalBench: A Collaboratively Built Benchmark for Measuring Legal Reasoning in Large Language Models}, 
+            author={Neel Guha and Julian Nyarko and Daniel E. Ho and Christopher Ré and Adam Chilton and Aditya Narayana and Alex Chohlas-Wood and Austin Peters and Brandon Waldon and Daniel N. Rockmore and Diego Zambrano and Dmitry Talisman and Enam Hoque and Faiz Surani and Frank Fagan and Galit Sarfaty and Gregory M. Dickinson and Haggai Porat and Jason Hegland and Jessica Wu and Joe Nudell and Joel Niklaus and John Nay and Jonathan H. Choi and Kevin Tobia and Margaret Hagan and Megan Ma and Michael Livermore and Nikon Rasumov-Rahe and Nils Holzenberger and Noam Kolt and Peter Henderson and Sean Rehaag and Sharad Goel and Shang Gao and Spencer Williams and Sunny Gandhi and Tom Zur and Varun Iyer and Zehua Li},
+            year={2023},
+            eprint={2308.11462},
+            archivePrefix={arXiv},
+            primaryClass={cs.CL}
+        },
+        """,
+        n_samples={"test": 300},
+        avg_character_length={"test": 103.21},
+    )
+
+    def dataset_transform(self):
+        mapping = {"yes": 1, "no": 0}
+        self.dataset = self.dataset.map(
+            lambda example: {
+                "answer": mapping.get(example["answer"].lower(), example["answer"])
+            }
+        )
+        self.dataset = self.dataset.rename_column("answer", "label")
+
+        self.dataset = self.dataset.map(
+            lambda example: {
+                "text": example["text"]
+                + "\n\n"
+                + "Parties are diverse: "
+                + example["parties_are_diverse"]
+                + "\n\n"
+                + "Amount-in-controversy is met: "
+                + example["aic_is_met"]
+            }
```

Comment:
This construction seems a bit odd to me. Seems like both are boolean values which can be reformulate into more readable plaintext

Reply:
We could formulate it to be of the form `The Amount-in-controversy was met.` / `The Amount-in-controversy was not met.`, based on the boolean value. Similarly, `The parties are diverse.`/ `The parties are not diverse.` for the other value as well. These sentences would be just appended after the text (without the newlines).

Reply:
I would prefer that - let us check how it influence performance though (guessing it is positive)

Reply:
The results do not seem to vary significantly, though there is a noticeable reduction in evaluation times.